praw/models/reddit/subreddit.py

```diff
@@ -1364,6 +1374,13 @@ def update(
             valid emoji string, for example ``':snoo:'``.
         :param max_emojis: (int) Maximum emojis in the flair
             (Reddit defaults this value to 10).
+        :param fetch: Whether or not PRAW will fetch existing information on
+            the existing flair before updating (Default: False).
+
+        .. warning:: If parameter ``fetch`` is set to False, a network request
+            to Reddit will not be made, but all other values will be
+            overwritten to their defaults. Furthermore, text and CSS class will
+            be set to blank values.
```

Comment:
I think this phrasing could be cleaned up. Consider something along the lines of

    If parameter ``fetch`` is set to ``False``, all parameters
    not provided will be reset to default (``None`` or ``False``) values.

Reply:
This is much better and more concise, thanks!